Data/Base.rte/Shaders/Blit8.frag

```diff
@@ -6,9 +6,9 @@ in vec2 textureUV;
 out vec4 FragColor;
 
 uniform sampler2D rteTexture;
-uniform sampler1D rtePalette;
```

Comment:
Why? Are 1d textures unsupported on some platforms or something?

Reply:
I'm guessing for gles support? Anything else should have support for tex1d, though I'm guessing driver implementation is probably mostly same as this here.

Reply:
Yes